src/utils/openOrdex.js

```diff
@@ -213,43 +215,53 @@ export async function generatePSBTListingInscriptionForBuy({
     receiverAddress,
     price,
     paymentUtxos,
-    dummyUtxo,
+    dummyUtxos,
     sellerSignedPsbt,
     inscription,
 }) {
     const psbt = new bitcoin.Psbt({ network: NETWORK });
 
     let totalValue = 0;
     let totalPaymentValue = 0;
+    let totalDummysValue = 0;
+
+    // Inputs:
+    // (# 0) - 600 sat dummy output, provided by taker
+    // (# 1) - 600 sat dummy output, provided by taker
+    for (let i = 0; i < 2; i++) {
```

Comment:
Cleaner if this is implemented as dummyUtxos.forEach() right?

Or should use dummyUtxos.length. Hardcoded "2" not recommended